api/src/backend/api/tests/test_models.py

```diff
@@ -1,6 +1,30 @@
 import pytest
+from allauth.socialaccount.models import SocialApp
+from django.core.exceptions import ValidationError
 
-from api.models import Resource, ResourceTag
+from api.db_router import MainRouter
+from api.models import (
+    Resource,
+    ResourceTag,
+    SAMLConfigurations,
+    SAMLDomainIndex,
+    Tenant,
+)
+
+VALID_METADATA = """<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Could we move this within the TestSAMLConfigurationModel class? It should only be used there, right?